tests/virt_autotest/switch_to_ssh_and_install_hypervisor.pm

```diff
@@ -50,9 +50,21 @@ sub run {
     zypper_call('--gpg-auto-import-keys ref');
     zypper_call("in -t pattern ${hypervisor}_server ${hypervisor}_tools", 1800);
     set_serial_console_on_vh('', '', $hypervisor);
+    systemctl 'enable libvirtd', ignore_failure => 1;
 
     virt_autotest::utils::install_default_packages();
     save_screenshot;
+
+    #set up necessary network bridge br0 for virtualization tests.
+    #SLE sets up br0 during installation as kvm/xen role, while openSUSE TW does not.
+    #change MACAddressPolicy to none, or the network bridge will get a different MAC address from its enslave
+    #it is specific to TW because it uses udev naming scheme v247, see bsc#1136600
+    my $bridge_rule_file = "/usr/lib/systemd/network/98-default-bridge.link";
+    assert_script_run("echo \"[Match]\nDriver=bridge\n\n[Link]\nMACAddressPolicy=none\" > $bridge_rule_file", 60);
+    my $default_netdevice = get_var('SUT_NETDEVICE', 'eno1');
```

Comment:
Why en01 is the last default value ?

Reply:
'eno1' in TW is like 'em1' in SLE. It happens that all my three test machines are using 'on board network card'.

Reply:
Why are different nics here and in above file? eno1 vs   enp1s0? Do they both exist on tw host, or either one?

Reply:
'eno1' is default interface for TW host, 'enp1s0' is for TW guest in TW kvm host. 'eth0' is for TW guest in TW XEN host.